assets/src/application/ui/layout/entityList/filterBar/filterState/useEntityListFilterStateManager.ts

```diff
@@ -19,86 +19,74 @@ const useEntityListFilterStateManager = <SortBy = BasicSortBy>(defaultSortBy: So
 	};
 	const [state, dispatch] = useReducer(getReducer<SortBy>(), initialState);
 
-	const setSortBy: FSM['setSortBy'] = useCallback(
-		(sortBy) => {
-			dispatch({
-				type: 'SET_SORT_BY',
-				sortBy,
-			});
-		},
-		[dispatch]
-	);
+	const setSortBy: FSM['setSortBy'] = useCallback((sortBy) => {
+		dispatch({
+			type: 'SET_SORT_BY',
+			sortBy,
+		});
+	}, []);
 
-	const setPerPage: FSM['setPerPage'] = useCallback(
-		(newPageNumber, newPerPage) => {
-			// the pagination component will recalculate the page number
-			// if it goes out of range after changing the perPage value,
-			// so save that else we'll get no results returned
-			if (newPageNumber && newPageNumber !== state.pageNumber) {
-				setPageNumber(newPageNumber);
-			}
-			dispatch({
-				type: 'SET_PER_PAGE',
-				perPage: newPerPage,
-			});
-		},
-		[dispatch]
-	);
+	const setPerPage: FSM['setPerPage'] = useCallback((newPageNumber, newPerPage) => {
+		// the pagination component will recalculate the page number
+		// if it goes out of range after changing the perPage value,
+		// so save that else we'll get no results returned
+		if (newPageNumber && newPageNumber !== state.pageNumber) {
+			setPageNumber(newPageNumber);
+		}
+		dispatch({
+			type: 'SET_PER_PAGE',
+			perPage: newPerPage,
+		});
+	}, []);
 
-	const setPageNumber: FSM['setPageNumber'] = useCallback(
-		(pageNumber) => {
-			dispatch({
-				type: 'SET_PAGE_NUMBER',
-				pageNumber,
-			});
-		},
-		[dispatch]
-	);
+	const setPageNumber: FSM['setPageNumber'] = useCallback((pageNumber) => {
+		dispatch({
+			type: 'SET_PAGE_NUMBER',
+			pageNumber,
+		});
+	}, []);
 
-	const setTotal: FSM['setTotal'] = useCallback(
-		(total) => {
-			dispatch({
-				type: 'SET_TOTAL',
-				total,
-			});
-		},
-		[dispatch]
-	);
+	const setTotal: FSM['setTotal'] = useCallback((total) => {
+		dispatch({
+			type: 'SET_TOTAL',
+			total,
+		});
+	}, []);
 
 	const setCardView: FSM['setCardView'] = useCallback(() => {
 		dispatch({
 			type: 'SET_VIEW',
 			view: 'card',
 		});
-	}, [dispatch]);
+	}, []);
 
 	const setTableView: FSM['setTableView'] = useCallback(() => {
 		dispatch({
 			type: 'SET_VIEW',
 			view: 'table',
 		});
-	}, [dispatch]);
```

Comment:
I have removed `dispatch` from deps. Since it never changes, it doesn't matter adding it to deps.